packages/terra-application-header-layout/examples/Index.site-page.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions */
```

Comment:
Moved from `terra-framework-site/src/examples/application-header-layout` & update to use the `IndexPageTemplate`